tutorials/hyperspectral/Work-With-Hyperspectral-Data-In-R.Rmd

```diff
@@ -211,60 +213,62 @@ that covers 800 nm-805 nm might have a FWHM of 2.5 and a wavelength value of 802
 
 The HDF5 dataset that we are working with in this activity contains more 
 information than we need to work with. For example, we don't necessarily need 
-to process all 426 bands - if we are interested in creating a product like NDVI
-which only users bands in the near infra-red and red portions of the spectrum. 
-Or we might only be interested in a spatial subset of the data - perhaps a region 
-where we have plots in the field.
+to process all 426 bands available in a full NEON hyperspectral reflectance file 
```

Comment:
Should this change now that it is already a reduced file with fewer bands?

Reply:
Good point. I changed this to read:
"For example, we don't necessarily need 
to process all 107 bands available in this example dataset (or all 426 bands 
available in a full NEON hyperspectral reflectance file, for that matter)
- if we are interested in creating a product like NDVI..."